src/utils/__tests__/fixtures/incident.json

```diff
@@ -70,6 +70,7 @@
       "openbare_ruimte": "Marnixstraat",
       "huisnummer_toevoeging": ""
     },
+    "area_code": "north",
```

Comment:
Will an incident always have this prop? Also for records in the database prior to this change?

Reply:
I think not. Shall I create an `incidentWithAreaCode.json` or something?

Reply:
I think it's no problem though, having the property there.